lib/teams/emrContainersRbacConfig.ytpl

```diff
@@ -0,0 +1,41 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
What was missing in the default role? Depending on what it is, we can decide whether to update the default or keep this one.

Reply:
The EMR on EKS define their own RBAC. This role is not used by the team but by the EMR on EKS Service Linked Role see the link below. The service linked role has a k8s user called `emr-containers`.

https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-cluster-access.html

Reply:
> The EMR on EKS define their own RBAC. This role is not used by the team but by the EMR on EKS Service Linked Role see the link below. The service linked role has a k8s user called `emr-containers`.
> 
> https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/setting-up-cluster-access.html

Got it, all clear now.